### DISCUSSION TITLE
For neovim users

### DISCUSSION THREAD

**Discussion by williambdean** [watercooler-labs/toggl-cli#85] [Show and tell]
This tool has been working like a breeze! 🙏 

I made a small wrapper around it to use from neovim: https://github.com/wd60622/toggl.nvim

**Comment by shantanuraj** [comment]
That's super cool I've been [working on the idea](https://github.com/watercooler-labs/toggl-cli/tree/shantanu/edit) of allowing time-entry edits in the cli using your `$EDITOR` (for me that's nvim)
so I really like the idea of making it working there, lmk if I can help you out with anything.

**Reply by williambdean** [reply]
Nice! Can you open PR (even if its in draft)

**Reply by shantanuraj** [reply]
I've opened a draft here https://github.com/watercooler-labs/toggl-cli/pull/86
